plugins/web/opentelemetry-instrumentation-document-load/src/types.ts

```diff
@@ -35,4 +35,6 @@ export interface DocumentLoadInstrumentationConfig
     documentFetch?: DocumentLoadCustomAttributeFunction;
     resourceFetch?: ResourceFetchCustomAttributeFunction;
   };
+  ignoreNetworkEvents?: boolean;
+  ignorePerformanceEvents?: boolean;
```

Comment:
Please add a section in this package README.md to document these options for package users, like the one for [`applyCustomAttributesOnSpan`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/plugins/web/opentelemetry-instrumentation-document-load/README.md#optional--add-custom-attributes-to-spans-if-needed) We can also include a comment here to help users understand what each option does and when and why to use it. Similar to the comment few lines above the `applyCustomAttributesOnSpan?:` option.